lib/ansible/modules/system/selinux.py

```diff
@@ -59,6 +61,34 @@
     state: disabled
 '''
 
+RETURN = '''
+msg:
+    description: Messages that describe changes that were made
+    returned: always
+    type: string
+    sample: Config SELinux state changed from 'disabled' to 'permissive'
+configfile:
+    description: Path to SElinux configuration file
+    returned: always
+    type: string
+    sample: /etc/selinux/config
+policy:
+    description: Name of the SELinux policy
```

Comment:
inconsistent 'SELinux' vs 'SElinux' (capital L in first, I'm not sure which is correct)

Reply:
SELinux is the correct capitalization. Fixed.